event-streams/jmxtrans/README.md

```diff
@@ -0,0 +1,182 @@
+
+## JmxTrans Configuration
+
+[JmxTrans documentation](https://github.com/jmxtrans/jmxtrans/wiki) provides further guidance on defining [queries](https://github.com/jmxtrans/jmxtrans/wiki/Queries) and [output writers](https://github.com/jmxtrans/jmxtrans/wiki/OutputWriters). The configuration depends largely on what information is to be captured and where it must be sent. 
+
+The configuration can be deployed as a ConfigMap and then made available to the JmxTrans tool. An example ConfigMap definition is included as a reference. 
+
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: <jmxtrans-config>
+  namespace: <namespace>
+  labels:
+    app.kubernetes.io/name: jmx-trans
+data:
+  config.json: |
+    '{
+      "servers": [
+        {
+          "host": "<eventstreams-name>-kafka-<broker-id>.<eventstreams-name>-kafka-brokers",
+          "port": 9999,
+          "queries": [
+            {
+              "obj": "kafka.server:type=BrokerTopicMetrics,name=*",
+              "attr": [
+                "Count"
+              ],
+              "outputWriters": [
+                {
+                  "@class": "com.googlecode.jmxtrans.model.output.StdOutWriter"
+                }
+              ]
+            }
+          ]
+        },
+        // repeat for each broker
+      ]
+    }'
+```
+
+Where:
+- `<jmxtrans-config>` is the name of the ConfigMap.
+- `<namespace>` is the namespace where Event Streams is installed.
+- `<eventstreams-name>` is the name of the Event Streams instance.
+- `<broker-id>` is Kafka broker ID (for example 0).
+
+
+## JmxTrans container image
+
+JmxTrans is a Java application. This means that a container image, which includes a Java runtime and the JmxTrans tool, is needed to deploy the application on Kubernetes environments. Build a custom image that include a Java runtime and the JmxTrans tool.
+
+**Important:** JmxTrans only supports Java 11.
+
+**Note:** Event Streams 11.1.6 (and earlier) included a container image for running the JmxTrans tool. For example, `cp.icr.io/cp/ibm-eventstreams-jmxtrans@sha256:c43965013d22d3227cf7c1bf1211d59b0595ce6d535ab3fab8e7c80694417f76`. This container image is no longer maintained and does not receive security updates. 
+
+### Custom JmxTrans image
+
+Build a container image that includes Java 11 runtime and the JmxTrans tool. An example [Dockerfile](./Dockerfile) to build such an image is provided as a reference. 
+
+You can build and push the container image by using these commands:  
+```
+docker build -t <image-registry>/<image-name>:<image-tag> .
+docker push <image-registry>/<image-name>:<image-tag>
+```
+
+## JmxTrans Deployment
+
+Create a deployment with the JmxTrans image and mount the configuration that is defined in the ConfigMap. An example deployment definition is included as a reference.
```

Comment:
I would like to indicate this is an example on how to do it.. and not like a specific instruction they must follow, like we do in our docs.. The reason we are putting this info here instead of our documentation is that this is not really a supported capability.

Reply:
updated my suggestion.